src/containers/Homepage/index.tsx

```diff
@@ -95,19 +96,21 @@ export class Homepage extends React.Component<HomepageProps, HomepageState> {
 
   editBlogButton(blog: IBlog): JSX.Element {
     return (
+      <Tooltip title={`edit Blog ${blog._id}`} onOpen={() => this.makeEditBlogSection(blog)}>
```

Comment:
Please start a tooltip with a capital letter... Edit Blog (not edit Blog), thanks